npm-shrinkwrap.json

```diff
@@ -4029,7 +4058,7 @@
       }
     },
     "webpack-core": {
-      "version": "0.6.8",
+      "version": "0.6.9",
```

Comment:
It's this guy: https://github.com/mzgoddard/hard-source-webpack-plugin/blob/master/package.json#L50